server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -719,6 +719,11 @@ public class ShuffleServerConf extends RssBaseConf {
           .booleanType()
           .defaultValue(false)
           .withDescription("Whether to enable app detail log");
+  public static final ConfigOption<Boolean> SERVER_WITH_MEMORY_STORAGE_TYPE_OPTIMIZE_ENABLED =
```

Comment:
Could we use camel style for last segment and 4 - 5 segments for the whole config option like
`rss.server.storage.bitmapMemoryOptimizeEnabled`.

Reply:
done